flopy/modflow/mfsfr2.py

```diff
@@ -928,7 +923,7 @@ def load(cls, f, model, nper=None, gwt=False, nsol=1, ext_unit_dict=None):
                         # dataset 6d description suggests that this line isn't read for
                         # isfropt > 1 but description of icalc suggest that icalc=2
                         # (8-point channel) can be used with any isfropt
-                        if i == 0 or nstrm > 0 and not reachinput or isfropt <= 1:
+                        if i == 0 or (nstrm > 0 and not reachinput) or isfropt <= 1:
```

Comment:
FYI, this one was an autofix for [RUF021](https://docs.astral.sh/ruff/rules/parenthesize-chained-operators/)